aries-site/src/examples/templates/forms/FilterExample.js

```diff
@@ -77,7 +77,7 @@ export const FilterExample = () => {
               />
             </FormField>
             <FormField
-              htmlFor="locationType"
+              htmlFor="locationType__input"
```

Comment:
Nice! How did you know to add the `__input`?

Reply:
Inspecting the DOM:
![Screen Shot 2020-03-11 at 12 02 01 PM](https://user-images.githubusercontent.com/1756948/76448545-224af680-6390-11ea-987e-e138a5bd4351.png)

Feels a bit brittle though. If Grommet were ever to change its implementation of how select/drop work, I think this would be susceptible of breaking.

Reply:
Gotcha -- yeah it would be interesting to see if needs to be better implemented on Grommet's side. I feel like users shouldn't need to inspect the DOM

Reply:
I'm fine with this for now though since it's creating a solution with how Grommet is currently implemented.